examples/emit.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const Emittery = require('../');
+
+class MyEmitter extends Emittery {}
```

Comment:
Would be nice to use this of an example where you don't need to extend Emittery.